src/Interfaces.ts

```diff
@@ -1,72 +1,136 @@
-export interface Claim<T extends ClaimAttributes = ClaimAttributes> {
+import * as Joi from 'joi'
+import JSIG = require('jsonld-signatures')
+
+const jsigs = JSIG()
```

Comment:
We shouldn't be instantiating JSIG in Po.et JS - whatever depends on the instance of JSIG should go inside the object that wraps JSIG